src/components/EditFeatureDrawer/EditFeatureGeometryToolbar/index.tsx

```diff
@@ -221,10 +222,16 @@ export const EditFeatureGeometryToolbar: React.FC<EditFeatureGeometryToolbarProp
         className="geometry-edit-tb"
         alignment="vertical"
       >
-        <ToggleGroup>
+        <ToggleGroup
+        selected={selected}
+        onChange={(_evt: any, value: string | undefined) => {
+          setSelected(value)
+        }}
+        >
           {
             allowedEditMode.includes('CREATE') ?
               <DrawButton
+                value="one"
```

Comment:
IMHO a more speaking value like "CREATE" would be better. Same for the buttons below.